<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>952</width>
    <height>804</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="echo_frame">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>411</width>
     <height>621</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_choose_model">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Load File</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="browse_file_input">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="browse_file">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_widget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>261</width>
      <height>351</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Folder</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>File Name</string>
     </property>
    </column>
   </widget>
   <widget class="QLineEdit" name="text_echo_filename">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>520</y>
      <width>271</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="echo_filename">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Echo Filename</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_echo_btn">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>470</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Echos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="show_plot_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Plot Graphs</string>
    </property>
   </widget>
   <widget class="QPushButton" name="show_echo_btn">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>470</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Plot Echos</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>User Interface for Data Analysis</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
